.github/ISSUE_TEMPLATE/Software-Project-Contribution.md

```diff
@@ -75,87 +96,83 @@ This is a list of questions that need to be answered by the contributor in order
 
 
 ### Target Contributors
-*Describe the contributor profile (background, position, organization) you would like to get contributions from.*
+*Describe the contributor profile (background, position, etc) and if applicable organizational profile (financial institutions, tech vendors, consulting firms, etc) you would like to get contributions from*
 
-## Project Communication Channel(s)
+## Preferred project infrastructure
+If the project is approved, FINOS will provision the infrastructure necessary for [development, release and project collaboration](https://community.finos.org/docs/collaboration-infrastructure). Below a list of questions which will help us accelerate the onboarding process and tailor it to the maintainers needs:
 
-- [ ] Contributor to ask maintainers which communications channels they'd like to use:
-- Asynchronous
-  - [ ] GitHub Issues _(public)_
-  - [ ] GitHub Discussions _(public)_
+- Which code hosting platform does the project prefer? *[Github](https://github.com/finos/)|[GitLab](https://gitlab.com/finos)* 
+- Please check below which asynchronous collaboration channels will the project like to use?
+  - [ ] GitHub Issues
+  - [ ] GitHub Public Discussions
   - [ ] GitHub Team Discussions _(consisting of the above described contributors)_
-    - [ ] Public
-    - [ ] Private
-  - [ ] Mailing-list (groups.io)
-  - [ ] FINOS Slack Channel (consisting of the above described contributors)
-    - [ ] General (public) _(supply channel name)_ 
-    - [ ] Leadership (private) _(supply channel name)_
+    - Public (Mandatory) _(consisting of the above described contributors)_
+    - [ ] Private (Optional)
+  - [ ] Mailing-list (lists.finos.org)
+    - General (Mandatory, for project community): `add preferred address, XXXX-general@lists.finos.org`
```

Comment:
I would suggest not making Mailing Lists and Slack Channels mandatory. We want to focus activity as much as possible on GitHub and use mailing lists and slack as additions when needed. We also don't want to build potential dead channels that we will need to monitor in case of activity and close in case of inactivity.